samples/app.overlay

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2024, CATIE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ #include <zephyr/dt-bindings/input/input-event-codes.h>
+ #include <zephyr/dt-bindings/lvgl/lvgl.h>
+ #include <zephyr/dt-bindings/gpio/sixtron-header.h>
+ #include <zephyr/dt-bindings/display/ili9xxx.h>
+ 
+  &timers1 {
+	 status = "okay";
+	 st,prescaler = <1000>;
+ };
+ 
+ / {
+	 aliases {
+		 pwm-dev-0 = &pwm1;
+	 };
+	 pwm_devices {
+		 status = "okay";
+		 compatible = "pwm-leds";
+ 
+		 backlight_lcd: pwm_dev_0 {
+			 status = "okay";
+			 pwms = <&pwm1 3 2000000 PWM_POLARITY_NORMAL>;
+		 };
+	 };
+ };
+ 
+ / {
+	 chosen {
+		 zephyr,display = &ili9163c0;
+	 };
+ 
+	 mipi_dbi {
+		 compatible = "zephyr,mipi-dbi-spi";
+		 dc-gpios = <&sixtron_connector DIO2 GPIO_ACTIVE_HIGH>;  /* Data/Command */
+		 spi-dev = <&sixtron_spi>;
+		 #address-cells = <1>;
+		 #size-cells = <0>;
+		 write-only;
+ 
+		 ili9163c0: ili9163c@0 {
+			 compatible = "ilitek,ili9163c";
+			 mipi-max-frequency = <20000000>;  // 20MHz
+			 reg = <0>;
+			 pixel-format = <1>;
+			 width = <128>;  
+			 height = <160>;  
+			 rotation = <90>;
+			 madctl = [00];
+			 frmctr1 = [08 08];
+			 pgamctrl = [3F 25 1C 1E 20 12 2A 90 24 11 00 00 00 00 00];
+			 ngamctrl = [20 20 20 20 05 00 15 A7 3D 18 25 2A 2B 2B 3A];
+			 pwctrl1 = [0A 02];
+			 pwctrl2 = [02];
+			 vmctrl1 = [55 60];
+			 vmctrl2 = [45];
+			 gamset = [04];  
+			 gamadj = [01];    
+			 dispinv = [07];    
```

Comment:
I couldn't find these values when I developed the Mbed OS driver, where did you find them?